trunk/SUAVE/Attributes/Solids/Acrylic.py

```diff
@@ -0,0 +1,29 @@
+# Acrylic.py
+#
+# Created: Jul 2017, J. Smart
+
+#-------------------------------------------------------------------------------
+# Imports
+#-------------------------------------------------------------------------------
+
+from Solid import Solid
+from SUAVE.Core import Data
+
+#-------------------------------------------------------------------------------
+# Acrylic Solid Class
+#-------------------------------------------------------------------------------
+
+class Acrylic(Solid):
+
+    """ Physical Constants Specific to Acrylic"""
+
+    def __defaults__(self):
+
+        self.UTS        = 75e6          # Ultimate Tensile Strength
```

Comment:
Since these variables can be used outside of the class they must have verbose names.

Reply:
Changed to e.g. ultimate_tensile_strength